VotingApplication/VotingApplication.Web/Views/Routes/UpDownVote.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿<div ng-controller="UpDownVoteController" ng-app="VotingApp">
+    <h3>Options</h3>
+    <div ng-repeat="option in options">
+        <div class="option-block">
+            <p class="option-text">{{option.Name}}</p>
+            <div class="option-buttons">
+                <span class="btn-group-updown">
+                    <button class="inactive-btn pull-left" ng-class="{'active-btn': option.voteValue==-1}" ng-click="(option.voteValue=(option.voteValue!=-1?-1:0))">
+                        <span class="glyphicon glyphicon-thumbs-down" aria-hidden="true"></span>
+                    </button>
+                    <button class="inactive-btn pull-right" ng-class="{'active-btn': option.voteValue==1}" ng-click="(option.voteValue=(option.voteValue!=1?1:0))">
+                        <span class="glyphicon glyphicon-thumbs-up" aria-hidden="true"></span>
```

Comment:
The up and down thumbs look like they're both one button instead of two buttons together

Also, can we use the Grey400 border instead of the Green500 one?